tests/test_enterprise/api_client/test_enterprise.py

```diff
@@ -111,42 +77,6 @@ def test_skip_request_if_response_cached(self):
         response = client._load_data('resource')  # pylint: disable=protected-access
         assert response == cache_value
 
-    @responses.activate
-    @mock.patch('enterprise.api_client.lms.JwtBuilder', mock.Mock())
-    @mock.patch('enterprise.api_client.discovery.get_edx_api_data', mock.Mock())
-    @mock.patch('enterprise.api_client.discovery.JwtBuilder', mock.Mock())
-    def test_get_content_metadata(self):
```

Comment:
same, i think this test is still relevant because the get_content_metadata method still exists and hits the enterprise catalog endpoints (https://github.com/edx/edx-enterprise/blob/2622a599b0d839821ee68d8b0a706bef3cccde1d/enterprise/api_client/enterprise.py#L33)

Reply:
We already have [test_get_content_metadata_with_enterprise_catalogs](https://github.com/edx/edx-enterprise/blob/master/tests/test_enterprise/api_client/test_enterprise.py#L152) to test `get_content_metadata` with `enterprise_catalogs`.